src/components/NavBar.tsx

```diff
@@ -14,49 +13,70 @@ function NavBar({ setActivePage }: NavBarActions) {
 
   return (
     <>
-      <button className="hamburger_container" onClick={() => (isShowingLinks ? setIsShowingLinks(false) : null)}>
+      <button
+        className="fixed flex flex-col justify-center items-center top-0 w-full font-semibold bg-transparent border-0"
+        onClick={() => (isShowingLinks ? setIsShowingLinks(false) : null)}
+      >
         <div
           onClick={() => setIsShowingLinks((status) => !status)}
-          className={`hamburger ${isShowingLinks ? 'hidden opacity-0' : 'block'}`}
+          className={`absolute right-0 top-0 p-2 cursor-pointer duration-500 transform rotate-180 hover:bg-hover_opaque md:opacity-0 md:hidden ${
+            isShowingLinks ? 'hidden opacity-0' : 'block'
+          }`}
         >
-          <span className="line w-1"></span>
-          <span className="line w-2"></span>
-          <span className="line w-3"></span>
+          <span className="block h-1 m-1 rounded-md bg-cream w-4"></span>
+          <span className="block h-1 m-1 rounded-md bg-cream w-5"></span>
+          <span className="block h-1 m-1 rounded-md bg-cream w-6"></span>
         </div>
       </button>
 
-      <nav className={`nav_bar ${isShowingLinks ? 'active_nav_bar' : ''}`}>
+      <nav
+        className={`top-0 h-full animate-slideInNav duration-150 text-cream font-semibold fixed w-48 hidden md:absolute md:flex md:justify-between md:w-full ${
```

Comment:
The sidebar never shows for me. I think you want to move this `hidden` into the else path of the `isShowingLinks ?` ternary on line 36
```suggestion
        className={`top-0 h-full animate-slideInNav duration-150 text-cream font-semibold fixed w-48 md:absolute md:flex md:justify-between md:w-full ${
```

I will say writing responsive styles look _awful_ in vanilla TW. Looking forward to see all this move into Twin Macro & styled components 😵‍💫

Reply:
Nav Bars are always tricky for me and i've written a few of them. On top of the tailwind refactor I guess this just slipped my mind. Thanks a ton for finding this!